src/utils/date/format.ts

```diff
@@ -0,0 +1,13 @@
+import { default as fnsFormat } from "date-fns/format";
```

Comment:
just copy the two methods youre using from date-fns from their github in the next story, its not worth it to install a lib with a hundred methods to just use two in the end.

Reply:
I use a lot of functions from `date-fns`. No only here